### DISCUSSION TITLE
Where To Obtain A Preconfigured Virtualbox VM For Retronas To Use With Windows 11 Host?

### DISCUSSION THREAD

**Discussion by SamuelBanya** [retronas/discussions#119] [General]
Hey there,

I found this which appears to be the virtual machine examples for RetroNAS:
- https://retro.ramonddevrede.nl/retronas/

However these are VMWare and Hyper V based, so I am not sure how to use them or how I can modify them to point to Central Timezone USA based sources for dependencies.

My goal is to deploy this in a Virtualbox VM on a Windows 11 host. This would be running on a Beelink S12 mini PC that would be only attached to a network switch downstairs without internet, each of those network switches would then be connected to my old school Windows 98 and XP machines so I can somehow connect SAMBA to the mini PC running Windows 11, and to use for LAN parties at my house.

**Comment by SamuelBanya** [comment]
Did some more research on this and it looks like I can just import this VM via Hyper-V manager in Windows 11:
- https://youtu.be/L2FaiuJrxwo?si=u8daJBBeL8pb3j16

Never knew about this as I have only used Virtualbox on Windows for Linux VMs and QEMU with virt-manager on Linux for VMs.

Will try to use this and see if that works, though I am not sure how I can change the apt source list to point from Europe to Central Timezone (USA)

**Comment by sairuk** [comment]
We provide retronas, it is an ansible based installer playground that runs _on_ linux. We have no interest in taking on shipping of preconfigured VMs images etc  when all preconfigured really means for retronas is the script is installed on a base linux install. If the community wants to provide that, awesome have at it.

That said however one member of the communty did already provide some hyperv stuff, and another was working on an rpi distro image but both are out of date by now. You could also probably just use WSL these days but I do not use windows so couldn't say.

There is plenty of documentation out there on how to configure everything in linux, if you want additional control over the underlying os with a ui installing the cockpit tool may be advantageous for you.
